infra/modules/providers/azure/data-factory/tests/unit/data_factory_unit_test.go

```diff
@@ -0,0 +1,84 @@
+package unit
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+	tests "github.com/microsoft/cobalt/infra/modules/providers/azure/data-factory/tests"
+	"github.com/microsoft/terratest-abstraction/unit"
+)
+
+var workspace = "pwc-" + strings.ToLower(random.UniqueId())
```

Comment:
What is the meaning of the `pwc` prefix? If we omit the workspace all-together, the test harness should generate a random one for you.

This would involve deleting this line + line 70